node/src/chain_spec.rs

```diff
@@ -82,6 +82,8 @@ pub struct Extensions {
     pub fork_blocks: sc_client_api::ForkBlocks<Block>,
     /// Known bad block hashes.
     pub bad_blocks: sc_client_api::BadBlocks<Block>,
+    /// The light sync state extension used by the sync-state rpc.
+    pub light_sync_state: sc_sync_state_rpc::LightSyncStateExtension,
```

Comment:
What is the use of this new variable?

Reply:
if they support Aura or we switch back to Babe in future when we figure out what was causing the timeout when querying the block header then that extension would be for light clients to connect to our chain. for example if using software that provided light client support similar to https://github.com/paritytech/substrate-light-ui